contracts/src/StakeTable.sol

```diff
@@ -17,7 +15,7 @@ using EdOnBN254 for EdOnBN254.EdOnBN254Point;
 /// @title Ethereum L1 component of the Espresso Global Confirmation Layer (GCL) stake table.
 ///
 /// @dev All functions are marked as virtual so that future upgrades can override them.
-contract StakeTable is Initializable, InitializedAt, OwnableUpgradeable, UUPSUpgradeable {
+contract StakeTable is Initializable, InitializedAt, UUPSUpgradeable {
```

Comment:
I wonder why not keep this contract as is and set the timelock as owner. When I read the doc string on the contract it sounds to me like this is an intended use case.

https://github.com/OpenZeppelin/openzeppelin-contracts/blob/0a77e54c307d9becf0473b73541c4f9d3b2743a3/contracts/governance/TimelockController.sol#L12

Reply:
ah i meant it as the owner and swapped onlyOwner() for onlyTimelock(), but you're right, i should be using the existing Ownable libraries,  i'll switch to that, thank you!

Reply:
https://github.com/EspressoSystems/espresso-network/pull/2846/commits/b3c1b2f79fbb549bd1f75e6b13884e53c1609375